apps/docs/pages/guides/functions/quickstart.mdx

```diff
@@ -106,6 +106,33 @@ const { data, error } = await supabase.functions.invoke('hello-world', {
 
 After invoking your Edge Function you should see the response `{ "message":"Hello Functions!" }`.
 
+### Error Handling
+
+When interacting with Edge Functions, it's important to be prepared for potential errors that might occur during the invocation. To handle errors effectively, the following error handling code can be used:
+
+```js
+import { FunctionsHttpError, FunctionsRelayError, FunctionsFetchError } from "@supabase/supabase-js";
+
+const { data, error } = await supabase.functions.invoke('hello', {
+  headers: {
+    "my-custom-header": 'my-custom-header-value'
+  },
+  body: { foo: 'bar' }      
+})
+
+if (error instanceof FunctionsHttpError) {
+  const errorMessage = await error.context.json()
+  console.log('Function returned an error', errorMessage)
```

Comment:
minor: use `console.error`?

Reply:
I think @kevcodez resolved this console.error.

Reply:
I didn't - it's still console.log right now